config/settings/base.py

```diff
@@ -67,3 +77,39 @@
 
 
 STATIC_URL = '/static/'
+
+REST_FRAMEWORK = {
+    # Use hyperlinked styles by default.
+    'DEFAULT_MODEL_SERIALIZER_CLASS':
+        'rest_framework.serializers.ModelSerializer',
```

Comment:
The comment `# Use hyperlinked styles by default.` does not agree with the actual default that you have configured ( `'rest_framework.serializers.ModelSerializer'` ).